conf/app.routes

```diff
@@ -3,6 +3,7 @@
 GET         /assets/*file                         controllers.Assets.at(path="/public", file)
 
 GET         /unauthorized                        @uk.gov.hmrc.tariffclassificationfrontend.controllers.SecurityController.unauthorized()
+GET         /case-not-found                      @uk.gov.hmrc.tariffclassificationfrontend.controllers.CaseController.caseNotFound(reference: String)
```

Comment:
Why do we need this?
We used to just show a not found view at the same endpoint